contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -44,6 +46,13 @@ library ValidationLogic {
     require(amount != 0, Errors.VL_INVALID_AMOUNT);
     require(isActive, Errors.VL_NO_ACTIVE_RESERVE);
     require(!isFrozen, Errors.VL_RESERVE_FROZEN);
+    require(
+      IERC20(reserve.aTokenAddress)
```

Comment:
change this condition: check if the supply cap is 0 (unset) || total supply < supply cap